workflows/iuf/operations/vcs-update/vcs-update-working-branch.yaml

```diff
@@ -283,4 +293,11 @@ spec:
         image: artifactory.algol60.net/csm-docker/stable/docker.io/portainer/kubectl-shell:latest-v1.21.1-amd64
         command: [bash]
         source: |
-          kubectl -n argo delete secret/{{inputs.parameters.vcs_user_credentials_secret_name}}
+          vcs_secret_name={{inputs.parameters.vcs_user_credentials_secret_name}}
```

Comment:
note there is a line up above `args: ["-c", "/opt/csm/cf-gitea-update/entrypoint.sh"]`. we need to audit the scripts in all these references (not just this one).  these scripts contain much of the messages we care about.

Reply:
Yes @JohnNavitskyHPE. We have identified around 10 files that are invoked from IUF templates. Will share that list with you and discuss upon that. We can take it up as a separate task in the current initiative and work on it. We can keep this PR specific for IUF templates and close it if we don't have any additional comments.